Chapter_5/emr/README.md

```diff
@@ -1,55 +1,75 @@
-### Setting up an EMR Cluster for ETL
+### EMR Setup
+ 
```

Comment:
A short introduction describing "Setting up an EMR Cluster for ETL"

Reply:
done